addons/account/models/account_account_tag.py

```diff
@@ -78,3 +86,24 @@ def _unlink_except_master_tags(self):
             master_tag = self.env.ref(f"account.{master_xmlid}", raise_if_not_found=False)
             if master_tag and master_tag in self:
                 raise UserError(_("You cannot delete this account tag (%s), it is used on the chart of account definition.", master_tag.name))
+
+    def _translate_tax_tags(self, langs=None, tag_ids=None):
+        """Translate tax tags having the same name as report lines."""
+        langs = langs or (code for code, _name in self.env['res.lang'].get_installed() if code != 'en_US')
+        for lang in langs:
+            self.env.cr.execute(SQL(
+                """
+                UPDATE account_account_tag tag
+                   SET name = tag.name || jsonb_build_object(%(lang)s, substring(tag.name->>'en_US' FOR 1) || (report_line.name->>%(lang)s))
+                  FROM account_report_expression expression
+                  JOIN account_report_line report_line ON report_line.id = expression.report_line_id
```

Comment:
```suggestion
                  FROM account_report_line report_line 
```
You don't need this join on account_report_expression, do you ? :thinking:  You're anyway matching the tags using the report line's name.

Reply:
I need to get the report to filter on the country, to replicate `_get_tax_tags_domain` as close as possible